source/speakers/speakers.md

```diff
@@ -2,70 +2,72 @@
 title: Listing yourself as a Potential Speaker
 ---
 
-If you're an Apache committer, and you're interested in speaking about Apache 
+If you're an Apache committer interested in speaking about Apache 
 Technologies, Apache Projects or the Apache Software Foundation as a whole, 
-we'd encourage you to get involved and list yourself in our speaker 
+we encourage you to get involved and list yourself in our speaker 
 application!
 
 ## Finding Speakers
 
-If you are organising an event and looking for speakers, please see
+If you are organising an event and looking for speakers, see
 [the Apache Speakers homepage](/speakers/) for details.
 
-## Listing yourself as a Potential Speaker
+## Listing yourself as a potential speaker
 
-To list yourself as someone interested in being an Apache Speaker, there are
-two steps involved. The first is to create a FOAF file for yourself, including
+To list yourself as someone interested in being an Apache speaker, there are
+two steps. The first is to create a FOAF ("friend of a friend") file for yourself that includes
 a couple of key pieces of information. The second is to list yourself in the
-speaking DOAP file.
+speaking DOAP ("description of a project") file.
 
 ### Creating your FOAF file
 
-Creating a FOAF file for yourself is very easy, assuming you don't already 
-have one available. Details on how to create a FOAF file, along with a
+If you do not already have a FOAF file for yourself, creating one is very easy. Details on how to create a FOAF file, along with a
 handy wizard to guide you through the process, are available at
-[https://people.apache.org/foaf/](https://people.apache.org/foaf/) .
+[https://people.apache.org/foaf/](https://people.apache.org/foaf/).
 
-When creating your FOAF file, as a potential speaker we do require you
-to include a few key bits of information. These are:
+Since you are a potential speaker we do require you
+to include a few key bits of information in the FOAF file. These are:
 
  * Your location
- * The projects you are involved in (include foundation wide ones too if you're
+ * The projects you are involved in (include foundation-wide ones, too, if you're
    happy to talk about these)
- * Your twitter ID (so we can point people at your speaking history on 
+ * Your Twitter ID (so we can point people at your speaking history on 
    [Lanyrd](http://lanyrd.com/))
 
 
 ### Listing yourself in the Speakers DOAP file
 
-Now, if you haven't already done so, checkout the committers directory from
+If you haven't already done so, check out the committers' directory from
```

Comment:
```suggestion
If you haven't already done so, check out the committers directory from
```

This references the `committers/` directory, so `committers` instead of `committers'`